distribution/lib/Standard/Base/0.0.0-dev/src/Data/Range.enso

```diff
@@ -127,6 +127,30 @@ type Range
             if self.step < 0 then go (<=) self.start else
                 throw_zero_step_error
 
+    ## PRIVATE
+       Applies a function to each element of the range.
+
+       Essentially acts like `range.to_vector.each_with_index`, but it is more
+       efficient.
+
+       Arguments:
+       - function: A function to apply that takes two indices: first the index
```

Comment:
You probably meant two parameters rather than indices?

Reply:
Well actually both of these parameters are an index of some kind. But I think the phrasing will be clearer indeed if I use parameters here. So I'll change it.